tests/sentry/migrations/test_0779_remove_groups_from_group_inbox.py

```diff
@@ -31,6 +33,7 @@ def setup_before_migration(self, app):
         GroupInbox.objects.create(group=self.ignored_group, reason=GroupInboxReason.NEW.value)
         GroupInbox.objects.create(group=self.unresolved_group, reason=GroupInboxReason.NEW.value)
 
+    @pytest.mark.skip(reason="unneeded")
```

Comment:
maybe I can just fully delete this, I don't think we need old migrations tests do we?